### DISCUSSION TITLE
Nested scroll support (horizontal)

### DISCUSSION THREAD

**Discussion by j2esu** [patrykandpatrick/vico#722] [Ideas]
Hi, do you have any plans of supporting nested scroll? It could be quite helpful if a scrollable CartesianChartHost, for example, is put inside of a HorizontalPager (compose)

**Comment by j2esu** [comment]
It lools that currently all the scroll delta is consumed even if the chart is not actually scrolling (overscrolling). I see the extra delta is used to move markers, and it definitely makes sense, but some may want to use it for nested scrolling / overscroll effect.

Possible implementation
https://github.com/patrykandpatrick/vico/pull/730

**Comment by Gowsky** [comment]
Hello! Do you plan to update your PR #730? If not, we'll look into this.

**Reply by j2esu** [reply]
Hi! Thanks for reviewing my PR and your meaningful comments. I did plan to try, but not sure if I can do that in the near future. Sorry for that, please feel free to look into this.

**Comment by Gowsky** [comment]
[Vico v2.0.0-alpha.22](https://patrykandpatrick.com/vico/releases/2.0.0-alpha.22/) addresses this. Cheers!
